### ISSUE TITLE
a11y heading level issues

### ISSUE THREAD

**Issue by susanev** [chef/automate#3843]
## overview
weve got a few places where we skip heading levels, lets fix it and make our heading levels accessible

## details
- [ ] infrastructure/client-runs h4 in modal, should be h2
- [ ] /compliance/reports/overview h4 in modal, should be h2
- [ ] scan-jobs/jobs h4 in errors for job, investigate whether this should be h2 or p
- [ ] apps empty state is an h3, it should be a p
